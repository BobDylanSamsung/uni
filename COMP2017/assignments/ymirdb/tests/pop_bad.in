set a 1 2 3
set b a
pop c
pop d
pop a
pop b
pop b
pop 1a
list entries
bye
