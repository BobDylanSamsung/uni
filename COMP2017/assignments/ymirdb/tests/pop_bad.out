ok

ok

no such key

no such key

1

a

nil

no such key

b []
a [2 3]

bye
